Sources/_StringProcessing/ConsumerInterface.swift

```diff
@@ -545,7 +554,7 @@ extension Unicode.POSIXProperty {
 extension Unicode.ExtendedGeneralCategory {
   // FIXME: Semantic level
   func generateConsumer(
-    _ opts: CharacterClass.MatchLevel
+    _ opts: MatchingOptionSet
```

Comment:
Is there any practical difference between the type that does stack logic and the this type?